packages/base/src/manager-base.ts

```diff
@@ -248,7 +248,7 @@ abstract class ManagerBase<T> {
         let commPromise;
         // we check to make sure the view information is provided, to help catch
         // backwards incompatibility errors.
-        if (!options.view_name || !options.view_module || !options.view_module_version) {
+        if (_.contains([options.view_name, options.view_module, options.view_module_version], undefined)) {
```

Comment:
Would you mind using the more standard Javascript construct:

```javascript
[options.view_name, options.view_module, options.view_module_version].indexOf(undefined) > 0
```
We only have one other place where we use underscore in this file, and I think it's better to use standard javascript these days where it isn't too much more work.